libbeat/outputs/elasticsearch/client.go

```diff
@@ -417,17 +417,18 @@ func (client *Client) bulkCollectPublishFails(result eslegclient.BulkResult, dat
 				encodedEvent := data[i].EncodedEvent.(*encodedEvent)
 				if encodedEvent.deadLetter {
 					stats.nonIndexable++
-					client.log.Errorf("Can't deliver to dead letter index event (status=%v). Enable debug logs to view the event and cause.", status)
-					client.log.Debugf("Can't deliver to dead letter index event %#v (status=%v): %s", data[i], status, msg)
+					client.log.Errorf("Can't deliver to dead letter index event (status=%v). Look at the event log to view the event and cause.", status)
+					client.log.Errorw(fmt.Sprintf("Can't deliver to dead letter index event %#v (status=%v): %s", data[i], status, msg), "log.type", "event")
 					// poison pill - this will clog the pipeline if the underlying failure is non transient.
 				} else if client.deadLetterIndex != "" {
-					client.log.Warnf("Cannot index event (status=%v), trying dead letter index. Enable debug logs to view the event and cause.", status)
-					client.log.Debugf("Cannot index event %#v (status=%v): %s, trying dead letter index", data[i], status, msg)
+					client.log.Warnf("Cannot index event (status=%v), trying dead letter index. Look at the event log to view the event and cause.", status)
+					client.log.Debugw(fmt.Sprintf("Cannot index event %#v (status=%v): %s, trying dead letter index", data[i], status, msg), "log.type", "event")
```

Comment:
Why in this case we log with debug level? In the default configuration the event would not be written in a log file, right?

Reply:
Yes, it should have been warn level. I fixed it.